src/main/java/core/basesyntax/dao/DataFromCsvFilrImpl.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.ReadFile;
+import core.basesyntax.db.ReadFileImpl;
+import core.basesyntax.service.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataFromCsvFilrImpl implements DataFromCsvFile {
+    public static final String STRING_TO_ARRAY_SEPARATOR = ",";
+    private static final ReadFile readFile = new ReadFileImpl();
+    private final List<FruitTransaction> fruitList = new ArrayList<>();
+
+    @Override
+    public List<FruitTransaction> readFileToList(String fileCsvPath) {
+        List<String> listFromFile = readFile.readFileToList(fileCsvPath);
```

Comment:
I think it better to make this method to work with List<String> listFromFile as method input parameter, and call  `readFile.readFileToList(fileCsvPath)` in main method